CHANGELOG.md

```diff
@@ -20,6 +22,7 @@
     FIXES [#3000](https://github.com/microsoft/Microsoft365DSC/issues/3000)
 * SPOTenantSettings
   * Add support for the CommentsOnSitePagesDisabled parameter.
+
```

Comment:
Please remove spacing in changelog to keep all release related items together.

Reply:
done